extractor/src/extractors/ChartYamlExtractor.ts

```diff
@@ -0,0 +1,51 @@
+import Extractor from "./Extractor";
+import ExtractorFile from "./ExtractorFile";
+import {ManifestDependency, ManifestFile} from "@depscloud/api/v1beta";
+import MatchConfig from "../matcher/MatchConfig";
+import Languages from "./Languages";
+
+export default class ChartYamlExtractor implements Extractor {
+    public async extract(url: string, files: { [p: string]: ExtractorFile }): Promise<ManifestFile> {
+        const {
+            name,
+            version,
+            dependencies: deps,
+        } = files["Chart.yaml"].yaml();
+
+        const dependencies: Array<ManifestDependency> = (deps || []).map((dep) => {
+            let fullName = String(dep.repository || "").trim();
```

Comment:
I'm wondering how we should include the repository here... it kinda makes sense. At the same time, it's harder to connect some of the deployment dots. For example. All I really care about is my project depends on redis & MySQL. The fact that the Bitnami chart might be a dependency acts more as a version qualifier than an actual identifying qualifier.